pkg/webhook/validation/uservalidation.go

```diff
@@ -57,13 +59,16 @@ func ValidateUserForFleetCRD(req admission.Request, whiteListedUsers []string, g
 }
 
 // ValidateUserForResource checks to see if user is allowed to modify argued resource modified by request.
-func ValidateUserForResource(req admission.Request, whiteListedUsers []string) admission.Response {
+func ValidateUserForResource(req admission.Request, whiteListedUsers []string, isTerminalCall bool) admission.Response {
```

Comment:
We call this function here https://github.com/Azure/fleet/blob/main/pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook.go#L151 where it's we don't reject immediately we call another function to allow/reject hence added the isTerminalCall variable to enable counting the rejection correctly